app/controllers/users_controller.rb

```diff
@@ -113,6 +113,18 @@ def extlogout
     render :extlogout, :layout => 'login'
   end
 
+  def test_mail
+    user = find_resource(:edit_users)
+    begin
+      MailNotification[:tester].deliver(:user => user, :email => params[:user_email])
+    rescue Net::SMTPAuthenticationError, Net::SMTPServerBusy, Net::SMTPSyntaxError, Net::SMTPFatalError, Net::SMTPUnknownError, Timeout::Error,
+           Errno::ECONNRESET, Errno::ECONNABORTED, Errno::EPIPE, Errno::ETIMEDOUT => e
+      Foreman::Logging.exception("Unable to deliver email", e)
+      render :json => e.message, :status => :unprocessable_entity and return
```

Comment:
Alternatively, keep the list here and create another catch for all the rest, but the code will be the same.